src/Robo/Commands/Drupal/DrupalCommand.php

```diff
@@ -20,14 +21,16 @@ class DrupalCommand extends BltTasks {
    */
   public function install() {
 
+    $account_name = 'super-user-' . RandomString::string(10);
```

Comment:
I suppose that if someone can review and index of all the usernames, then it poses an issue.